src/components/search-paneset/search-paneset.js

```diff
@@ -48,13 +57,41 @@ export default class SearchPaneset extends Component {
     this.state = {};
   }
 
+  createDeletePermissionName = 'ui-eholdings.titles-packages.create-delete';
+
+  isPackageOrTitle = (resultsType) => !!(resultsType === searchTypes.PACKAGES || resultsType === searchTypes.TITLES);
```

Comment:
I can agree with the last notice, that it will be more readable to convert into `resultsType !== searchTypes.PROVIDERS`. Good point, thanks!

But what about `!!` -> when you are using logic operator `||` it returns `true` or the last `false` expression. That's why it's better to convert into boolean.
For example: 
``` javascript
false || -5 // returns -5
```

Reply:
I can agree with the last notice, that it will be more readable to convert into resultsType !== searchTypes.PROVIDERS. Good point, thanks!

But what about !! -> when you are using logic operator || it returns true or the last false expression. That's why it's better to convert into boolean.
For example:
``` javascript
false || 0 // returns 0
!!(false || 0) / returns false
```

Reply:
Yep, but here you have just identity operator, it will return bool anyway
```
resultType === searchType.PACKAGES // bool
retultType === searchType.TITLES // bool 
bool || bool // bool
```